### PR TITLE
Add `rt.ht`

### PR DESCRIPTION
**Pull request by CetinSert**
Public Suffix List (PSL) Pull Request (PR) Template
====

Each PSL PR needs to have a description, rationale, indication of DNS validation and syntax checking, as well as a number of acknowledgements from the submitter.  This template must be included with each PR, and the submitting party MUST provide responses to all of the elements in order to be considered.

### Checklist of required steps

* [x] Description of Organization
* [x] Robust Reason for PSL Inclusion
* [x] DNS verification via dig
* [x] Run Syntax Checker (make test)

* [x] Each domain listed in the PRIVATE section has and shall maintain at least two years remaining on registration, and we shall keep the \_PSL txt record in place in the respective zone(s) in the affected section

__Submitter affirms the following:__ 

  * [x] We are listing *any* third-party limits that we seek to work around in our rationale such as those between IOS 14.5+ and Facebook (see [Issue #1245](https://github.com/publicsuffix/list/issues/1245) as a well-documented example)

  * [x] This request was _not_ submitted with the objective of working around other third-party limits

  * [x] The [Guidelines](https://github.com/publicsuffix/list/wiki/Guidelines) were carefully _read_ and _understood_, and this request conforms
  * [x] The submission follows the [guidelines](https://github.com/publicsuffix/list/wiki/Format) on formatting and sorting

---

For Private section requests that are submitting entries for domains that match their organization website's primary domain, please understand that this can have impacts that may not match the desired outcome and take a long time to rollback, if at all.

To ensure that requested changes are entirely intentional, make sure that you read the affectation and propagation expectations, that you understand them, and confirm this understanding. 

PR Rollbacks have lower priority, and the volunteers are unable to control when or if browsers or other parties using the PSL will refresh or update.

(Link: [about propagation/expectations](https://github.com/publicsuffix/list/wiki/Guidelines#appropriate-expectations-on-derivative-propagation-use-or-inclusion))

 * [x] *Yes, I understand*.  I could break my organization's website cookies etc. and the rollback timing, etc is acceptable.  *Proceed*.
---

Description of Organization
====

Organization Website: https://rtcode.io

RTCode.io is an Elefunc service that enables real-time web development and hosting. We currently provide users with `*.rt.ht` domains if they don't have their own.

I am the CEO/CTO of [Elefunc, Inc.](//elefunc.com) building [RTCode.io](//rtcode.io) and [RTEdge.net](//rtedge.net).

Reason for PSL Inclusion
====

* We issue SSL certificates for our customers.
* We host customer sites on Cloudflare.
* We want customer sites to be isolated from each other.
* We want to prevent setting cookies on the apex domain.

Number of users this request is being made to serve: 1200 monthly active users (current; growing).

DNS Verification via dig
=======

```
dig +short TXT _psl.rt.ht
"https://github.com/publicsuffix/list/pull/1860"
```

Results of Syntax Checker (`make test`)
=========

All tests passed.

### CODE REVIEW

**Comment by CetinSert** [comment]
Has there been any progress on this issue?
Please let us know if there are additional steps we need to take on our side to expedite the process of adding our domain to the PSL.

**Review by simon-friedberger — APPROVED** [review]
LGTM.
* [x] Tests pass.

**Comment by CetinSert** [comment]
Monthly reminder!
Still waiting on this.

**Comment by CetinSert** [comment]
Happy new year everyone!
I hope you guys get to make progress on this.

**Comment by CetinSert** [comment]
Finally some progress! (Watching this request daily.)

**Comment by CetinSert** [comment]
Waiting for your update!

**Comment by CetinSert** [comment]
@simon-friedberger
You merging https://github.com/publicsuffix/list/pull/1773/commits/3e98ecbea90c7c2a45709f1e45c4cf809f12b6e3 caused a conflict which I have gone ahead and resolved for you all in https://github.com/publicsuffix/list/pull/1860/commits/61f6d479052addca3172895e923897762178f28e.

Really looking forward to some real progress = merge on our pull request now! : )

File: public_suffix_list.dat [hunk 1]
```diff
@@ -12959,6 +12959,10 @@ easypanel.host
 // Submitted by <infracloudteam@namecheap.com>
 *.ewp.live
 
+// Elefunc, Inc. : https://elefunc.com
+// Submitted by Cetin Sert <cetin@elefunc.com>
```
Comment (simon-friedberger) [inline public_suffix_list.dat:12963]: Can you use a non-personal e-mail, please?

**Comment by simon-friedberger** [comment]
* [x] Expiration (Note: Must STAY >2y at all times)
  * [x] rt.ht expires 2026-05-10
* [x] DNS _psl entries (Note: Must STAY in place)
* [x] Tests pass
* [x] Sorting
* [x] Reasoning/Organization description

**Review by simon-friedberger — APPROVED** [review]

**Comment by CetinSert** [comment]
- ✔️ https://github.com/publicsuffix/list/pull/1860/commits/63b435e2584fd25b05d6a8583b4e0b9b0ff3859b changed to `domains@elefunc.com`

**Comment by simon-friedberger** [comment]
This has conflicts now, please update.

**Comment by CetinSert** [comment]
@simon-friedberger – I have just resolved the conflict!

**Pull request merged**